resources/views/competitions/partials/_form_competitions.blade.php

```diff
@@ -1,23 +1,18 @@
 @include('layouts.errors')
 
 <div class="form-item -padded">
-    {!! Form::label('campaign_id', 'Campaign ID:', ['class' => 'field-label']) !!}
-    {!! Form::text('campaign_id', (isset($room->campaign_id)) ? $room->campaign_id : NULL, ['class' => 'text-field']) !!}
+    {!! Form::label('contest_id', 'Contest ID:', ['class' => 'field-label']) !!}
+    {!! Form::text('contest_id', (isset($competition->contest_id)) ? $competition->contest_id : NULL, ['class' => 'text-field']) !!}
```

Comment:
@weerd this doesn't seem to work on these date fields when setting the default value. Am I doing something wrong here: 

```{!! Form::date('competition_start_date', $competition->competition_start_date or old($competition->competition_start_date), ['class' => 'text-field']) !!}```